UnitsNet.Serialization.JsonNet.Tests/UnitsNetBaseJsonConverterTest.cs

```diff
@@ -54,8 +54,7 @@ public void UnitsNetBaseJsonConverter_ConvertValueUnit_works_as_expected()
 
             Assert.NotNull(result);
             Assert.IsType<Power>(result);
-            Assert.True(Power.FromWatts(10.2365m).Equals((Power)result, 1E-5m, ComparisonType.Absolute));
-
+            Assert.True(Power.FromWatts(10.2365).Equals((Power)result, 1e-5, ComparisonType.Absolute));
```

Comment:
Duplicate the test case, so we test both double and decimal quantities.

Nitpick, but I would rather test `result.Value` and `result.Unit` instead of using `Equals()` in this test, to not mix too many implementations.

Reply:
Keeping it to have some tests still deserializing decimal values